concourse/scripts/tag-release.sh

```diff
@@ -20,17 +20,22 @@ Host github.com
   IdentityFile $(pwd)/git-key
 EOF
 
-version=$(cat release-version/number)
-tag="${TAG_PREFIX}${version}"
+cd paas-cf
+if git tag -l --contains HEAD | grep -E -q 'stage-\d\.\d\.\d'
```

Comment:
This script should be generic and not reference environment names directly. One simple approach is to create a new environment variable in the Makefile to control this behaviour, something like `GENERATE_RELEASE_VERSION`.
Or refactor the other variables `TAG_PREFIX` and `PAAS_CF_TAG_FILTER`.